tests/hardware_reboot/testdata/test_run.txt

```diff
@@ -0,0 +1,58 @@
+# Simple test of USB passthrough functionality after reboot of guest
+
+{{$usb_dev := "2-2"}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:ssh] stop
+
+eden pod deploy -n n1 --memory=512MB docker://itmoeve/eclient:0.3 -p 8027:22 --adapters USB2:2
+test eden.app.test -test.v -timewait 20m RUNNING n1
+
+exec -t 20m bash ssh.sh 8027
+stdout 'Ubuntu'
+
+exec -t 20m bash get-usb.sh 8027
+grep 'QEMU USB HARDDRIVE' {{$usb_dev}}.usb.product
+
+# restart pod
+eden pod stop n1
+exec sleep 20
+eden pod start n1
```

Comment:
How does this compare with the description of the PR? I mean "reboot **from the guest**".